web/generate-zones-config.js

```diff
@@ -44,6 +44,43 @@ const mergeExchanges = () => {
   return exchanges;
 };
 
+const mergeRatioParameters = () => {
+  // merge the fallbackZoneMixes, isLowCarbon, isRenewable params into a single object
```

Comment:
Is this something we might want to do directly in the config files?
Perhaps following a structure like this:
```yaml
overrides:
  2021:
    fallbackZoneMixes:
    isLowCarbon:
    isRenewable:
  2022:
    fallbackZoneMixes:
    isLowCarbon:
    isRenewable:
# ...
```
I suppose this could become a problem if the values don't change  at the same time though but it could be worth taking a look at at least.

Reply:
I'm not sure to understand what you have in mind here :)

Reply:
I might have misunderstood what this function does but if it just combines the fallbackZoneMixes, isLowCarbon and isRenewable that might be something we want to do directly in the config files so we can avoid this step. But I haven't really had the time look into it more in depth yet.

(This should not be a blocker for merging this PR tho.)

Reply:
It does so for all zonesOverrides, which now live in the respective zone configs. 
So unfortunately we can't really do it directly in a config file. 

I want to have a second look into this once everything is running smoothly to optimise the config flow. Ideally, we might not need that object. I had to add it due to backend-dependencies, but I'm pretty sure we could do things in a smoother way.

Reply:
Sounds like a plan then! 👍🏻 
I'll mark this as resolved.